server/controllers/userController.js

```diff
@@ -14,31 +16,41 @@ class UserController {
         where: { email: req.body.email }
       });
       if (foundUser) {
-        return util.errorstatus(res, 409, 'email address exist already');
+        return util.errorStatus(res, 409, 'email address exist already');
       }
 
       const hashPassword = auth.hashPassword(req.body.password);
+      const mailToken = Auth.generateMailToken({
+        firstName: req.body.firstName, 
+        email: req.body.email,
+        password: req.body.hashPassword
+      });
 
       const user = {
         firstName: req.body.firstName,
         lastName: req.body.lastName,
         email: req.body.email,
         password: hashPassword,
+        email_confirm_code: mailToken,
         signupMethod: 'local'
       };
-
       const token = auth.generateToken(user);
-
+    
       const createdUser = await models.Users.create(user);
-      return util.successStatus(res, 201, 'User added successfully', {
+      const link = `https://helobooks-staging.herokuapp.com/api/v1/auth/verifyEmail?token=${mailToken}`
```

Comment:
Hey bro. This is the staging link, Please use the production link 
https://helobooks.herokuapp.com/api/v1/